source/MaterialXView/Viewer.h

```diff
@@ -362,6 +417,10 @@ class Viewer : public ng::Screen
     bool _bakeAverage;
     bool _bakeOptimize;
     bool _bakeRequested;
+    
+    int _bakeWidth;
```

Comment:
Booleans and ints should be initialized, because it caused me hours of debugging previously.